src/index.js

```diff
@@ -150,8 +155,101 @@ const renderLevel = (root, levelNumber) => {
         .join("")}
     </tr>
   `;
+
+  renderSettings = () => {  
+    const settingsHTML = `
+    <div id="modal">
+      <div class="modal-content">
+        <span class="close material-symbols-outlined">cancel</span>
+        <h1>Setting</h1>
+      </div>
+    </div>
+    `;
+    document.querySelector("#overlay-screen").innerHTML = settingsHTML;
+
+    document.querySelector(".close").addEventListener("click", () => {
+      document.querySelector("#modal").style.display = "none";
+    });
+  };
+
+  renderHelp = () => {  
+    const helpHTML = `
+    <div id="modal">
+      <div class="modal-content">
+        <span class="close material-symbols-outlined">cancel</span>
+        <h1>How To Play</h1>
+        <p>Objective: Push the boxes to all of the goals to complete the level in the least amount of moves.</p>
+        <p>Use the mouse or keyboard to move around the board.</p>
+        <p>Keyboard Shortcuts:</br>
+          direction: arrow keys;  undo: z;  reset: r
+        </p>
+      </div>
+    </div>
+    `;
+    document.querySelector("#overlay-screen").innerHTML = helpHTML;
+
+    document.querySelector(".close").addEventListener("click", () => {
+      document.querySelector("#modal").style.display = "none";
+    });
+  };
+
+  renderLevelComplete = () => {
+    const completeHTML = `
+    <div id="modal">
+      <div class="modal-content"> 
+        <h1 class="text-glow">Level ${soko.levelNumber()+1}</h>
+        <h1 class="text-glow">Complete!</h1>
+        <table>
+          <tr>
+            <td>Current moves:</td>
+            <td id="current-move" class="score">${soko.movesSize()}</td>
+          </tr>
+          <tr>
+            <td>Best moves:</td>
+            <td id="best-move" class="score">${localStorage.getItem(soko.levelNumber()+1)}
+            </td>
+          </tr>
+        </table>
+        <div id="controls">
+          <button title="Random" id="random-level"><span class="material-symbols-outlined">shuffle</span></button>
+          <button title="Home" id="change-level"><span class="material-symbols-outlined">home</span></button>
+          <button title="Next" id="next"><span class="material-symbols-outlined">fast_forward</span></button>
```

Comment:
As a user, the "fast forward" button usually doesn't communicate "next", it communicates "skip ahead/to the end". I think there should be a better fit for "next", like a "play" button.

Reply:
When I'm on the last level and I press "next", nothing happens.

It'd be cool to be able to dismiss the "level complete" overlay modal so I can see the level I just finished.

Reply:
Ideally, if it's the last level, we can disable the next button. I don't know how to disable the button yet.

Reply:
I need help with disabling buttons. Can you implement one button and I can apply it on to the rest of the other buttons. Thank you

Reply:
Can we do that in a separate PR? Feel free to remind me if I forget or open an issue in https://github.com/ggorlen/cs195-project/issues if you don't mind.

Reply:
Yes, I need that code. Thank you. I think I can do this.